src/pages/resultsView/download/DownloadTab.tsx

```diff
@@ -204,6 +206,18 @@ export default class DownloadTab extends React.Component<IDownloadTabProps, {}>
         invoke:()=>Promise.resolve(this.alteredSamples.result!.join("\n"))
     });
 
+    readonly unalteredSamples = remoteData<string[]>({
```

Comment:
Give this a name that suggests what it returns.  I would guess Samples, but it actually returns sampleIds.   What I would actually do here is return the sample objects and then do the mapping to composite ID in business logic.

Reply:
Sure, will do!